src/main/java/htmlpublisher/HtmlPublisherTarget.java

```diff
@@ -148,7 +158,11 @@ public boolean getKeepAll() {
     }
 
     public boolean getAllowMissing() {
-           return this.allowMissing;
+        return this.allowMissing;
+    }
+
+    public boolean getOnlyCreateReportWithDifferentName() {
+        return this.onlyCreateReportWithDifferentName;
```

Comment:
This will need a null check done to make sure the default is returned if the field isn't set (i.e. is null). See `getEscapeUnderscores` for an example although obviously the default return will be different :)